stm32-modules/heater-shaker/firmware/system/startup_stm32f303xe.s

```diff
@@ -78,8 +78,25 @@ LoopCopyDataInit:
 	adds	r2, r0, r1
 	cmp	r2, r3
 	bcc	CopyDataInit
+	movs r1, #0
```

Comment:
IS this generated?

Reply:
The file in general? Part of it (the interrupt vector table part) probably is. These parts are probably handwritten. Specifically my changes here are copied from the data init section and altered a little bit to work the ccm copy.